Formula/irssi.rb

```diff
@@ -43,13 +46,13 @@ def install
       system "./autogen.sh", *args
     end
 
-    # https://github.com/irssi/irssi/pull/927
-    inreplace "configure", "^DUIfm", "^DUIifm"
-
     system "./configure", *args
     # "make" and "make install" must be done separately on some systems
     system "make"
     system "make", "install"
+
+    ENV["PERL5LIB"] = lib/"perl5/site_perl"
+    system "perl", "-e", "use Irssi"
```

Comment:
can you move the comments associated with these test lines up here too and delete them from the test block? perhaps also add a line about doing it in def install so that the logfiles have the relevant info about perl?

Reply:
aye